### DISCUSSION TITLE
Issues after updating micronaut-parent from 4.6.1 to 4.6.2

### DISCUSSION THREAD

**Discussion by rsalbergaria** [micronaut-projects/micronaut-platform#1798] [Q&A]
I recently updated the micronaut-parent version in my project from 4.6.1 to 4.6.2. After performing the update, I encountered several compilation errors when running mvn clean install. Below are some of the errors I am seeing:

> ProposalItemEntity.java:[21] error: cannot find symbol
  symbol:   class AuditEntityBuilder
  location: class AuditEntity
[ERROR] io.micronaut.inject.visitor.ElementPostponedToNextRoundException: Original element: inter.model.entity.ProposalAccountEntity$ProposalAccountEntityBuilderImpl is postponed to the next round!
        at io.micronaut.annotation.processing.visitor.JavaVisitorContext.checkForPostponedOriginalElement(JavaVisitorContext.java:358)
        at io.micronaut.annotation.processing.visitor.JavaVisitorContext.checkForPostponedOriginalElements(JavaVisitorContext.java:365)

**Comment by rsalbergaria** [comment]
@sdelamo @graemerocher Do you want to help me?

**Reply by dstepanov** [reply]
Do you have a custom type visitor? Can you post the full stacktrace

**Reply by graemerocher** [reply]
can you provide an example in a reproducer? Have you tried with 4.7.2?

**Reply by sdelamo** [reply]
@rsalbergaria try with [Micronaut Framework 4.7.1](https://github.com/micronaut-projects/micronaut-platform/releases/tag/v4.7.1) which is the latest stable release of Micronaut Framework 4. You should not find any breaking change from 4.6 to 4.7.1
